backend/wmg/data/extract.py

```diff
@@ -1,11 +1,23 @@
 # TODO generalize
+import os
+
+from backend.corpora.common.corpora_orm import DatasetArtifactFileType
+from backend.corpora.common.entities import Dataset, DatasetAsset
+from backend.corpora.common.utils.db_session import db_session_manager
+
+
 def get_s3_uris():
     """
     Retrieve list of s3 uris for datasets included in the wmg cube
     """
-    pass
+    with db_session_manager() as session:
+        dataset_ids = Dataset.list_ids_for_cube(session)
+        s3_uris = DatasetAsset.s3_uris_for_datasets(session, dataset_ids, DatasetArtifactFileType.H5AD)
+    return s3_uris.keys()
 
 
 def copy_datasets_to_instance(s3_uris, dataset_directory):
     """Copy given list of s3 uris to the provided path"""
-    pass
+    for dataset in s3_uris:
+        sync_command = f"aws s3 sync {s3_uris[dataset]} ./{dataset_directory}/{dataset}/local.h5ad"
+        os.subprocess(sync_command)  # TODO parallelize this step
```

Comment:
as far as I know boto3 doesnt support sync, its not completely necessary here since we are just copying single h5ad files but generally I've found sync to be faster and simpler particularly for tiledb objects. I think @ebezzi might also have made some changes to this function in his work to get the pipeline running on batch